cinema/serializers.py

```diff
@@ -1 +1,105 @@
-# write serializers here
+from rest_framework import serializers
+from .models import Genre, Movie, Actor, MovieSession, CinemaHall
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.CharField(
+        max_length=65,
+        read_only=True
+    )
```

Comment:
I really agree with most options, but they would fail tests )
I mean if we provide data in MovieView, without serializer which uses related to Actor Serializer, we can't provided actual data with method of Actor (get full name) which is requested in tests, so we can't simply made str return which I made in first variation of task. So I provide full_name field...

But, in case you were talking about read_only, I totally agree it's redundant, furthermore, since our max_length for first_name + last_name + 3 spaces == 515, I guess would be good to refactor max_length of this return accordingly.